functests/4_latency/latency.go

```diff
@@ -99,7 +99,20 @@ var _ = Describe("[performance] Latency Test", func() {
 		Expect(err).ToNot(HaveOccurred(), "error looking for the optional selector: %v", err)
 
 		Expect(workerRTNodes).ToNot(BeEmpty())
-		workerRTNode = &workerRTNodes[0]
+
+		//At least one worker node should have cpu.Allocatable greater than the quantity requested by each test, else skip the test
+		for _, wrkNode := range workerRTNodes {
+			allocatableCPU, _ := wrkNode.Status.Allocatable.Cpu().AsInt64()
+			Expect(err).ToNot(HaveOccurred(), "error getting allocatable cpus on the node: %v", err)
+			if allocatableCPU >= int64(latencyTestCpus) {
+				workerRTNode = &wrkNode
+				break
+			}
+		}
+		if workerRTNode == nil {
```

Comment:
Superfluous. we're already checking that it's not empty on line 101

Reply:
On line 101 we are checking that the list of the nodes is not empty, while here we are checking that we found a worker node with at least the same CPU quantity to run the test.